### PR TITLE
CheckNowCommandForm#init(): set submit label

### LINKED ISSUE #4281: Weird curl behavior when quering monitoring/service/show
## Describe the bug
A clear and concise description of what the issue is.

If i go to the following Icinga Page (Service Status page with added &format=json) https://ICINGAHOST/monitoring/service/show?host=HOST&service=SERVICE&format=json
i will see the complete Result in JSON format.

If i query the same url via curl, i got "{"status":"success","data":{"message":"Scheduling check.."}}" and the Check is being rescheduled.

## To Reproduce
Provide a link to a live example, or an unambiguous set of steps to reproduce this issue. Include configuration, logs, etc. to reproduce, if relevant.

curl to a monitoring service:

curl -k -u USER -v -H 'Accept: application/json' "https://ICINGAHOST/monitoring/service/show?host=HOST&service=SERVICE&format=json"

## Expected behavior
I want to get the same JSON Result if i query the url via curl.

## Screenshots
-

## Your Environment
Include as many relevant details about the environment you experienced the problem in

* Icinga Web 2 version and modules (System - About):  Icinga Web 2 Version 2.7.3
Modules 
director | 1.7.2
doc | 2.7.3
enforceddashboard | 0.0.0
grafana | 1.3.6
incubator | 0.5.0
ipl | v0.3.0
monitoring | 2.7.3
reactbundle | 0.7.0

* Web browser used: Chrome 84 
* Icinga 2 version used (`icinga2 --version`): r2.11.3-1
* PHP version used (`php --version`):  PHP 7.3.19-1
* Server operating system and version: Debian GNU/Linux 10 (buster)

### PR DESCRIPTION
**Pull request by Al2Klimov**
... otherwise Form#handleRequest() bypasses Form#isSubmitted()
on API requests and considers the CheckNowCommandForm submitted.

fixes #4281

### CODE REVIEW

**Comment by Al2Klimov** [comment]
## Before

#4281

## After

```
➜  icingaweb2 git:(bugfix/curl-monitoring-service-show-4281) curl -Lvu icingaadmin:icinga -H 'Accept: application/json' http://localhost:8080/icingaweb2/monitoring/service/show\?host\=test-random-2\&service\=ping4\&format\=json
*   Trying ::1...
* TCP_NODELAY set
* Connection failed
* connect to ::1 port 8080 failed: Connection refused
*   Trying 127.0.0.1...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 8080 (#0)
* Server auth using Basic with user 'icingaadmin'
> GET /icingaweb2/monitoring/service/show?host=test-random-2&service=ping4&format=json HTTP/1.1
> Host: localhost:8080
> Authorization: Basic aWNpbmdhYWRtaW46aWNpbmdh
> User-Agent: curl/7.64.1
> Accept: application/json
>
< HTTP/1.1 200 OK
< Date: Thu, 26 Nov 2020 17:44:20 GMT
< Server: Apache/2.4.6 (CentOS) OpenSSL/1.0.2k-fips
< X-Powered-By: PHP/7.3.11
< Transfer-Encoding: chunked
< Content-Type: application/json
<
* Connection #0 to host localhost left intact
{"status":"success","data":{"instance_name":"default","host_attempt":"1\/3","host_icon_image":"","host_icon_image_alt":"","host_acknowledged":"0","host_active_checks_enabled":"1","host_address":"127.0.0.1","host_address6":"","host_alias":"test-random-2","host_display_name":"test-random-2","host_handled":"0","host_in_downtime":"0","host_is_flapping":"0","host_last_state_change":"1603817356","host_name":"test-random-2","host_notifications_enabled":"1","host_passive_checks_enabled":"1","host_state":"0","host_state_type":"1","service_icon_image":"","service_icon_image_alt":"","service_acknowledged":"0","service_acknowledgement_type":"0","service_action_url":"","service_active_checks_enabled":"1","service_active_checks_enabled_changed":"0","service_attempt":"1\/5","service_check_command":"ping4","service_check_execution_time":"4.007753","service_check_interval":"60","service_check_latency":"0.000273","service_check_source":"localhost","service_check_timeperiod":null,"service_current_notification_number":"0","service_description":"ping4","service_display_name":"ping4","service_event_handler_enabled":"1","service_event_handler_enabled_changed":"0","service_flap_detection_enabled":"0","service_flap_detection_enabled_changed":"0","service_handled":"0","service_in_downtime":"0","service_is_flapping":"0","service_is_reachable":"1","service_last_check":"1606412597","service_last_notification":null,"service_last_state_change":"1596118734","service_long_output":"","service_next_check":"1606412656","service_next_update":"1606412726","service_notes":"","service_notes_url":"","service_notifications_enabled":"1","service_notifications_enabled_changed":"0","service_obsessing":"0","service_obsessing_changed":"0","service_output":"PING OK - Packet loss = 0%, RTA = 0.06 ms","service_passive_checks_enabled":"1","service_passive_checks_enabled_changed":"0","service_percent_state_change":"0","service_perfdata":"rta=0.057000ms;100.000000;200.000000;0.000000 pl=0%;5;15;0","service_process_perfdata":"1","service_state":"0","service_state_type":"1","vars":[],"contacts":[],"contact_groups":[],"servicegroups":{"ping":"Ping Checks"}}}* Closing connection 0
➜  icingaweb2 git:(bugfix/curl-monitoring-service-show-4281)
```

Oh, and yes, the respective button at http://localhost:8080/icingaweb2/monitoring/service/show?host=test-random-2&service=ping4 still works and is not triggered on page load.

**Review by nilmerg — APPROVED** [review]
This is actually caused by https://github.com/Icinga/icingaweb2/pull/2754/commits/235e75d054826d9dcac8db4240e031c167ea23f7.

Though, changing this now after so much time in a hopefully soon legacy form implementation leads only to more problems with forms requiring this flawed handling.

So..
